src/components/tabs/tabs.hbs

```diff
@@ -1,8 +1,8 @@
 <nav data-tabs class="{{@root.prefix}}--tabs" role="navigation">
   <div class="{{@root.prefix}}--tabs-trigger" tabindex="0">
     <a href="javascript:void(0)" class="{{@root.prefix}}--tabs-trigger-text" tabindex="-1"></a>
-    <svg width="10" height="5" viewBox="0 0 10 5">
-      <path d="M0 0l5 4.998L10 0z" fill-rule="evenodd" />
+    <svg width="12" height="7" viewBox="0 0 12 7">
```

Comment:
Should this caret to chevron change be applied to stable, too?

Reply:
Good catch. I updated it to have an if statement for different icons 👍